sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>nit: seems to be two spaces between <code>this</code>  and <code>changes</code>  in <code>Based on this  change the training configuration...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Reply:
<p>fixed</p>